distribution/docs/src/main/resources/content/_coreConcepts/attribute-stores.adoc

```diff
@@ -11,10 +11,10 @@
 |Standard
 |Support Status
 
-|https://tools.ietf.org/html/rfc4510[Lightweight Directory Access Protocol (LDAP/LDAPS)]
+a|https://tools.ietf.org/html/rfc4510[Lightweight Directory Access Protocol (LDAP/LDAPS)] {external-link}
```

Comment:
Is this a stray `a`

Reply:
nope, it's for formatting asciidoctor content within a table cell